library/ntc_show_command.py

```diff
@@ -303,7 +271,7 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             connection=dict(choices=['ssh', 'offline', 'netmiko_ssh',
-                            'trigger_ssh'], default='netmiko_ssh'),
+                            'trigger_ssh', 'netmiko_telnet'], default='netmiko_ssh'),
```

Comment:
Can you also add a `telnet` choice?  It'll be cleaner than using `netmiko_telnet` - we need/ed the distinction for ssh this one also supports trigger.